src/odemis/util/test/img_test.py

```diff
@@ -1181,6 +1181,40 @@ def test_rotate_img_metadata(self):
             numpy.testing.assert_array_equal(image, rotated_img)  # The data should not change, only the metadata.
 
 
+class TestFloodFill(unittest.TestCase):
+
+    def test_standard_fill(self):
+        """Test the flood_fill as it is expected to be used"""
+        a = numpy.zeros((7, 7), dtype=bool)
+        for i in range(min(a.shape)):
+            a[i, i] = True
+            a[-i - 1, i] = True
+
+        filled_array = img.apply_flood_fill(a, (3, 0))
+
+        self.assertTrue(filled_array[3, 1])
+        self.assertFalse(filled_array[3, -1])
+        self.assertFalse(filled_array[0, 3])
+        self.assertFalse(filled_array[-1, 3])
```

Comment:
I think you could do a more thorough check here. You know exactly what the filled array should look like, so you could just create the expected filled array and check that they are equal.

```python
expected_array = numpy.array([
       [1, 0, 0, 0, 0, 0, 1],
       [1, 1, 0, 0, 0, 1, 0],
       [1, 1, 1, 0, 1, 0, 0],
       [1, 1, 1, 1, 0, 0, 0],
       [1, 1, 1, 0, 1, 0, 0],
       [1, 1, 0, 0, 0, 1, 0],
       [1, 0, 0, 0, 0, 0, 1]
])
numpy.testing.assert_array_equal(expected_array, filled_array)
```